Assets/BossRoom/Scripts/Client/UI/Lobby/CreateLobbyUI.cs

```diff
@@ -0,0 +1,188 @@
+using System;
+using System.Text;
+using BossRoom.Scripts.Shared.Infrastructure;
+using BossRoom.Scripts.Shared.Net.UnityServices.Lobbies;
+using Unity.Multiplayer.Samples.BossRoom;
+using Unity.Multiplayer.Samples.BossRoom.Visual;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BossRoom.Scripts.Client.UI
+{
+
+    public class CreateLobbyUI : MonoBehaviour
+    {
+        public const string k_DefaultIP = "192.168.1.1";
+        public const int k_DefaultPort = 0;
+
+        private LobbyUIManager m_LobbyUIManager;
+
+        [SerializeField]
+        private InputField m_LobbyNameInputField;
+
+        [SerializeField]
+        private InputField m_IPInputField;
+
+        [SerializeField]
+        private InputField m_PortInputField;
+
+        private OnlineMode m_OnlineMode;
+
+        #region temp
+
+        [SerializeField]
+        [Tooltip("The Animating \"Connecting\" Image you want to animate to show the client is doing something")]
+        private GameObject m_LoadingImage;
```

Comment:
I'm not super familiar with Unity stuff, so maybe it's standard to call this m_LoadingImage.  I would call this loadingObject or something more generic, unless we're going to restrict this object to being an actual image object.